snap/snapcraft.yaml

```diff
@@ -0,0 +1,54 @@
+name: starbase
+version: git
+base: core24
+summary: Single-line elevator pitch for your amazing snap # 79 char long summary
+description: |
+  This is starbase description. You have a paragraph or two to tell the
+  most important story about your snap. Keep it under 100 words though,
+  we live in tweetspace and your description wants to look good in the snap
+  store.
+confinement: strict
+grade: devel
+license: GPL-3.0
+compression: lzo
```

Comment:
@lengau I am not sure about this but I understand that the snap taking a bit more space to get a faster start-up time is an interesting trade off for us.